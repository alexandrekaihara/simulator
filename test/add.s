        LOAD TEN
        ADD MINUSONE
        ADD MINUSNINE
        JMPZ ISZERO
            OUTPUT MINUSONE
            STOP
ISZERO: LOAD TEN
        ADD FIVE
        ADD MINUSFIFTEEN
        JMPZ ISZERO2
            OUTPUT MINUSONE
            STOP
ISZERO2: OUTPUT ONE
         STOP
TEN: CONST 10
MINUSONE: CONST -1
MINUSNINE: CONST -9
FIVE: CONST 5
MINUSFIFTEEN: CONST -15
ONE: CONST 1