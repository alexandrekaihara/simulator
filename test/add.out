10 25 1 26 1 27 8 11 13 26 14 10 25 1 28 1 29 8 22 13 26 14 13 30 14 10 -1 -9 5 -15 1